pkg/onboarding/application/common/helpers.go

```diff
@@ -0,0 +1,25 @@
+package common
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+)
+
+// HashString hashes the string into a bytes and casts it into a string.
+func HashString(s string) (string, error) {
+	if s == "" {
+		return "", fmt.Errorf("the string must not be empty")
+	}
+	h := sha256.New()
+
+	_, err := h.Write([]byte(s))
+	if err != nil {
+		return "", fmt.Errorf("an error has occurred while writing the byte objects")
+	}
+
+	bs := h.Sum(nil)
+
+	resp := hex.EncodeToString(bs)
+	return resp, nil
+}
```

Comment:
The hashing should be thorough 

We need to consider adding a salt, have _N_ number of  iterations and if possible do base64 encoding at the end. Kindly check our _PIN_ hashing algorithm